WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlow.swift

```diff
@@ -25,7 +26,17 @@ class BloggingRemindersFlow {
 
         NoticesDispatch.lock()
         bottomSheet.show(from: viewController)
-        UserDefaults.standard.setValue(true, forKey: Self.weeklyRemindersKeyPrefix + blog.objectID.uriRepresentation().absoluteString)
+        setHasShownWeeklyRemindersFlow(for: blog)
+    }
+
+    private static func hasShownWeeklyRemindersFlow(for blog: Blog) -> Bool {
+        let key = Self.weeklyRemindersKeyPrefix + blog.objectID.uriRepresentation().absoluteString
+        return UserDefaults.standard.bool(forKey: key)
+    }
+
+    private static func setHasShownWeeklyRemindersFlow(for blog: Blog) {
+        let key = Self.weeklyRemindersKeyPrefix + blog.objectID.uriRepresentation().absoluteString
```

Comment:
Nitpick, but we could have this key defined in a single static func for `userDefaultsKey(for:)`, but fine to leave it as is for now :)